apps/meteor/app/api/server/v1/moderation.ts

```diff
@@ -0,0 +1,240 @@
+import {
+	isReportHistoryProps,
+	isArchiveReportProps,
+	isReportInfoParams,
+	isReportMessageHistoryParams,
+	isModerationDeleteMsgHistoryParams,
+	isReportsByMsgIdParams,
+} from '@rocket.chat/rest-typings';
+import { ModerationReports, Users, Messages } from '@rocket.chat/models';
+import type { IModerationReport } from '@rocket.chat/core-typings';
+
+import { API } from '../api';
+import { deleteReportedMessages } from '../../../../server/lib/moderation/deleteReportedMessages';
+import { getPaginationItems } from '../helpers/getPaginationItems';
+
+type ReportMessage = Pick<IModerationReport, '_id' | 'message' | 'ts' | 'room'>;
+
+API.v1.addRoute(
+	'moderation.getReports',
```

Comment:
don't need the verb 'get' - `GET` is self evident. 

additionally, can you change the name to something more clear? Like `reportsByUsers`? to make it clear that this groups by users.

Reply:
Thank you, changed! I knew it; I am bad at naming 🙈